test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml

```diff
@@ -9,6 +9,7 @@ n_loaders: 1
 n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
+gce_instance_type_db: 'n1-highmem-32'
```

Comment:
`i3.4xlarge` has 16 vCPUs
shouldn't we be fitting the sizes?

Reply:
We wanted to test 24/32 cores for large instances that's why I put 32 here

Reply:
was it necessary for all tests? or just some specific ones?

Reply:
Changed to 16